proto/buf/alpha/registry/v1alpha1/display.proto

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/role.proto";
+
+// DisplayService is the display service that
+// provides information regarding to UI element display.
```

Comment:
```suggestion
// DisplayService provides information regarding UI element displaying
// based on the users role against a resource.
```

Reply:
`based on the users role against a resource`
is this necessary true? server and organization are probably not considered as a resource, but there will be display endpoints related to them later. Also I am trying to avoid using words related to authorization/role/permission to avoid confusing with other services, but it seems unavoidable.
how about just `based on the users roles`?